index.d.ts

```diff
@@ -122,7 +116,7 @@ export interface ResponsePromise extends Promise<Response> {
 	const result = await ky(…).json<Result>();
 	```
 	*/
-	json<T = JSONValue>(): Promise<T>;
+	json<T = unknown>(): Promise<T>;
```

Comment:
Having it default to `JSONValue` doesn't have much value as the whole point of having it strict is to ensure that the user-defined response type is only using JSON-supported types, but that's currently not possible. So you end up having to do `ky() as unknown as CustomType`. Better to leave it as `unknown` until https://github.com/microsoft/TypeScript/issues/15300 is fixed.